controllers/board-game-review-controllers.js

```diff
@@ -24,3 +23,13 @@ exports.getReviews = (req, res, next) => {
     });
 };
 
+exports.getReview = (req, res, next) => {
+    const id = req.params.review_id
+    Promise.all([checkIfReviewIdExists(id), isIdValid(id), selectReview(id)])
```

Comment:
You've added and extra connection to your db here which isn't completely necessary. You only want to check things are valid or exist when you need to. PSQL in this instance will tell you without having to open up other connections to dbs which can be an expensive process in time and cash

Reply:
So should I implement a PSQL error handler instead?